TIS-100_Commented/32050.0.txt

```diff
@@ -0,0 +1,51 @@
+@0
+# Solution reference: https://www.youtube.com/watch?v=mTgXCFsgnkk
```

Comment:
Would be better to solve the levels yourself of course, but it's good that you have provided references for the YouTube solutions you have used